.unreleased/template.release_notes.j2

```diff
@@ -0,0 +1,16 @@
+{#
+ # Variables:
+ #   release_current = the release version that has being builded
+ #   release_previous = previous release version
+ #   release_date = date of the release in format YYYY-MM-DD
+ #}
+## {{ release_current }} ({{ release_date }})
+
+This release contains performance improvements and bug fixes since 
+the {{ release_previous }} release. We recommend that you upgrade at the next
+available opportunity.
```

Comment:
Text probably needs to be slightly different depending on whether it is a patch release or a minor release and whether there are urgent fixes, something like:

- We recommend that you upgrade immediately
- We recommend that you upgrade at earliest available opportunity
- We recommend that you upgrade as next convenient opportunity

Reply:
Yes we can, but this is only a template file for reference. The unreleased release notes header will be placed in .unreleased/RELEASE_NOTES_HEADER.md.j2 where we'll really write the text that will be used for release PR and GH page. Have a look in other PR #6874.